### PR TITLE
For Nonprofits Page

### PR DESCRIPTION
**Pull request by pH-li**
Ticket: https://linear.app/uoft-blueprint-internal/issue/INTL-30/for-nonprofits-page

This code adds in the new for nonprofits page, which you can access by now clicking any "contact us" button.

### CODE REVIEW

File: src/app/for-nonprofits/page.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            Decision Criteria
+          </Title>
+        </Container>
+        <SimpleGrid
+          cols={2}
+          verticalSpacing={{ base: "md", sm: "sm" }
+          }
+        >
+          <div>
+            <Image
```
Comment (jordanjanakievski) [inline src/app/for-nonprofits/page.tsx]: This is not responsive. We will need to reassess the sizing for the image

File: src/app/for-nonprofits/page.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+              Although we want to help as many organizations as
+              possible, we can only work with a few nonprofits at
+              a time because of the amount of resources each
+              partnership requires. As such, we use the following
+              criteria to find projects with the best fit.
+            </Text>
+          </div>
+        </SimpleGrid>
+
+        <SimpleGrid
```
Comment (jordanjanakievski) [inline src/app/for-nonprofits/page.tsx]: These guys get a little squished and overflow when the viewport gets small

File: src/app/for-nonprofits/page.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        cols={3}
+        verticalSpacing={{ base: "lg", sm: "lg" }}
+        style={{margin:"5% 8% 5% 8%"}}>
+          <DecisionCard criteria={"Impact"} description={"How much will your project impact the community?"} image={"/nonprofit_page/Impact.svg"}/>
+          <DecisionCard criteria={"Necessity"} description={"How vital is your project to your non-profit organization?"} image={"/nonprofit_page/Necessity.svg"}/>
+          <DecisionCard criteria={"Scope"} description={"Is your project achievable within the allotted timeline?"} image={"/nonprofit_page/Scope.svg"}/>
+        </SimpleGrid>
+
+        <SimpleGrid
```
Comment (jordanjanakievski) [inline src/app/for-nonprofits/page.tsx:81]: Same as above

**Review by jordanjanakievski — COMMENTED** [review]
Just a couple comments. Mostly responsive. On the big screen it looks really great

File: src/app/for-nonprofits/page.tsx (hunk 2 above)
Comment (jordanjanakievski) [reply src/app/for-nonprofits/page.tsx]: We can vertically align the items when the viewports get to mobile. We can look at it together

File: src/app/for-nonprofits/page.tsx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+                  objectFit: "contain",
+                  position: "relative",
+                  alignItems: "end"}}
+                src="/about_page/placeholder.png" />
+            </center>
+          </div>
+          <div>
+            <center>
+              <Text
+                style={{color: "#0078e8",
```
Comment (jordanjanakievski) [inline src/app/for-nonprofits/page.tsx]: This text should get center formatted when we vertically align it with the image

**Review by jordanjanakievski — COMMENTED** [review]
Couple things. Should be quick. We can work on them in the meeting

**Comment by jordanjanakievski** [comment]
Screenshot 2024-12-14 at 17 17 16

Added some changes. Page is a bit more responsive and now has an actual image instead of temporary filler

**Review by sal3h-yas1in — APPROVED** [review]

**Pull request merged**